e2e-tests/conf/client.yml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ps-client
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: ps-client
+  template:
+    metadata:
+      labels:
+        name: ps-client
+    spec:
+      containers:
+      - name: ps-client
+        image: percona/percona-xtradb-cluster-operator:1.0.0-backup
```

Comment:
we will change it in the feature

Reply:
I'm using `percona/percona-server:8.0.25` for client